airbyte-test-utils/src/main/java/io/airbyte/test/annotations/IntegrationTest.java

```diff
@@ -1,21 +0,0 @@
-/*
```

Comment:
We need to remove the related gradle task as well. 
https://github.com/airbytehq/airbyte/blob/master/build.gradle#L374
and https://github.com/airbytehq/airbyte/blob/master/build.gradle#L387
as well as the tags name: https://github.com/airbytehq/airbyte/blob/master/build.gradle#L341